ledger-tool/src/main.rs

```diff
@@ -1765,6 +1765,20 @@ fn main() {
                         .help("Number of slots in the output"),
                 )
         )
+        .subcommand(
+            SubCommand::with_name("drop-cf")
+                .setting(AppSettings::Hidden)
```

Comment:
Thoughts on whether this should be hidden?

Reply:
Making it hidden looks good to me as this command is only used during major version rollback.  In addition, incorrectly using this command could make the ledger store unreadable or inconsistent.